convert2rhel/unit_tests/cert_test.py

```diff
@@ -98,30 +98,49 @@ def test_install_cert(self):
         shutil.rmtree(unit_tests.TMP_DIR)
 
 
+@pytest.fixture
+def remove_cert_setup(monkeypatch, tmpdir):
+    tmp_file = tmpdir / "filename"
+
+    monkeypatch.setattr(cert.SystemCert, "_get_cert", value=mock.Mock(return_value=("anything", "anything")))
+    monkeypatch.setattr(cert.SystemCert, "_get_target_cert_path", value=mock.Mock(return_value=str(tmp_file)))
+
+    sys_cert = cert.SystemCert()
+
+    return {
+        "cert_path_obj": tmp_file,
+        "sys_cert_instance": sys_cert,
+    }
```

Comment:
What do you think about moving this fixture to a `conftest.py` file? I think it can be well organized to separate the fixture from an actual test code.

Reply:
I think it's good to put a fixture into conftest.py when we think we can reuse it in other unit tests but we shouldn't put it there if it can only be used in a single unit test file.  In this case, I see that there are unittests in main_test.py that (once they are ported to pytest) could make use of this so it makes sense to move it to me.